src/components/DashboardHome/tests/MainContent.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { breakpoints } from '@edx/paragon';
+
+import '../../../__mocks__/reactResponsive.mock';
```

Comment:
This is the first time I have seen import this way, can we be more explicit here? otherwise it's ok.

Reply:
@mushtaqak Can you be a little more clear on what you mean by "more explicit"?